Code/Framework/AzCore/AzCore/Component/ComponentApplication.h

```diff
@@ -313,9 +308,6 @@ namespace AZ
         /// Common logic shared between the multiple Create(...) functions.
         void        CreateCommon();
 
-        /// Create the operating system allocator if not supplied in the StartupParameters
-        void        CreateOSAllocator();
-
         /// Create the system allocator using the data in the m_descriptor
         void        CreateSystemAllocator();
```

Comment:
Is the CreateSystemAllocator call still needed?

Reply:
This method still has some code to configure the allocator tracking of the SystemAllocator, so I renamed it.